docs/marketplace/dev/feature-integration-guides/202204.0/merchant-portal-feature-integration.md

```diff
@@ -18,7 +18,7 @@ See [Marketplace Merchant Portal Core feature integration](/docs/marketplace/dev
 
 ## Environment requirements
 
-- NodeJs v12+
+- Node.js v12+
 - Yarn v2 (or latest Yarn v1)
 - Spryker supported PHP version (7.3, 7.4)
```

Comment:
Release 202204.0 already requires "php": ">=7.4" as can be seen [here](https://github.com/spryker-feature/marketplace-merchantportal-core/blob/202204.0/composer.json#L9).